packages/react/scripts/generate-css.ts

```diff
@@ -0,0 +1,162 @@
+import path from 'path';
+
+import postcss from 'postcss';
+import postcssModules from 'postcss-modules';
+import glob from 'fast-glob';
+import fs from 'fs-extra';
+import * as R from 'ramda';
+
+import { generateScopedName } from './name';
+
+const outputFolder = path.resolve(__dirname, '../../css');
+fs.ensureDirSync(outputFolder);
+
+/** To exclude files when we start writing css manually */
+const ignore = [''];
+const additional = ['utility.module.css'];
+
+const findComponentName = (filePath: string) => {
+  let fileName = filePath.split('/src/')[1];
+
+  if (filePath.includes('utilities/')) {
+    fileName = fileName.split('utilities/')[1].split('/')[0];
+  }
+
+  if (fileName.includes('legacy/')) {
+    fileName = fileName.split('legacy/')[1].split('/')[0];
+  }
```

Comment:
I know this is a big edge case, and we can just update the script.
But would it be better to have a string array for these folder, that we can check, and then split on it?
Gives no speed boost, just makes it easy to modify

Reply:
Yeah, I was contemplating to rewrite this part aswell but left it for the future as I had already rewrote a lot, and we might not need to support more as this is only a temporary solution 😅 

Was thinking it can be done in the future if we need to do this for more folders. Would be have a lookup list somewhere and expand the `components` object to include an object with folder/group in addition to list of files thats today.